tardis/simulation/base.py

```diff
@@ -517,7 +517,7 @@ def log_plasma_state(
         plasma_state_log["next_w"] = next_dilution_factor
         plasma_state_log.columns.name = "Shell No."
 
-        if is_notebook():
+        if False and is_notebook(): #TODO: remove it with something better
```

Comment:
I think we can simply remove the `if` block since we are controlling `display` now